index.html

```diff
@@ -0,0 +1,423 @@
+                    <div class="container follower">
+                        <div class="follower-profile-img">
+                            <div class="follower-profile-img-pic six"></div>
+                        </div>
+                        <div class="follower-profile-info">
+                            <p>Spongebob</p>
+                            <span>@spongebobsquarepants</span>
+                        </div>
+                        <div class="follower-profile-more"><i class="fa-solid fa-user-plus"></i></div>
+                    </div>
+                    <div class="container follower">
+                        <div class="follower-profile-img">
+                            <div class="follower-profile-img-pic seven"></div>
+                        </div>
+                        <div class="follower-profile-info">
+                            <p>Patrick</p>
+                            <span>@patrick1337</span>
+                        </div>
+                        <div class="follower-profile-more"><i class="fa-solid fa-user-plus"></i></div>
+                    </div>
+                    <div class="container follower">
+                        <div class="follower-profile-img">
+                            <div class="follower-profile-img-pic four"></div>
+                        </div>
+                        <div class="follower-profile-info">
+                            <p>H2O</p>
+                            <span>@waterbot</span>
+                        </div>
+                        <div class="follower-profile-more"><i class="fa-solid fa-user-plus"></i></div>
+                    </div>
+                    <div class="container follower">
+                        <div class="follower-profile-img">
+                            <div class="follower-profile-img-pic five"></div>
+                        </div>
+                        <div class="follower-profile-info">
+                            <p>Paddy</p>
+                            <span>@leprechaun</span>
+                        </div>
+                        <div class="follower-profile-more"><i class="fa-solid fa-user-plus"></i></div>
+                    </div>
+                    <div class="container follower">
+                        <div class="follower-profile-img">
+                            <div class="follower-profile-img-pic two"></div>
+                        </div>
+                        <div class="follower-profile-info">
+                            <p>Fun Fact 101</p>
+                            <span>@funfactmachine</span>
+                        </div>
+                        <div class="follower-profile-more"><i class="fa-solid fa-user-plus"></i></div>
+                    </div>
+                    <div class="show-more">
+                        <a href="#">Show More</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </main>
+</body>
+</html>
```

Comment:
You can improve you class names using BEM metodology, for example home is parend element. home__main is children element of home and home__main-item is children element of home__main.

Reply:
That's definitely something I will have a look at, I thought my class names were a bit messy overall. Thanks for the insight! (p.s the section you highlighted was rushed last minute haha)